### ISSUE TITLE
Formly Fields - Text in the template field is un-tabbable with screenreaders

### ISSUE THREAD

**Issue by yasminmenchaca** [GSA/sam-design-system#1292]
**Is your feature request related to a problem? Please describe.**
When the user is utilizing JAWS or any screenreader within a Formly Form, they need to be able to know where they are on the page. If there is text with information and instructions, they should be able to tab to the template field text without multiple keys. This is within a form such as the Address Form example in the sam design system docs. https://gsa.github.io/sam-design-system/?path=/story/formly-examples-address-form--address-form

**Describe the desired solution**
If the template field text can be tabbed to, the user's screenreader can read said text on demand.

**Describe alternatives you've considered**
Unable to configure within Formly Configs.

**Additional context**
Video:
https://drive.google.com/file/d/1x866eNGpfVm_IDNKog3igbtfbFtBNuFf/view?usp=drive_link

Screenshots:
![Screenshot 2023-08-16 at 1 15 08 PM](https://github.com/GSA/sam-design-system/assets/57374092/d742a0d0-4f51-4ee2-8300-5bc9ec83f7e3)
![Screenshot 2023-08-16 at 1 16 35 PM](https://github.com/GSA/sam-design-system/assets/57374092/12fe5838-6bb3-46af-a5dd-6526fb0b96e6)
